src/test/java/api/requests/ItemsInTransitReportTests.java

```diff
@@ -469,6 +470,50 @@ void reportWillNotFailWithUriTooLargeError() {
     assertThat(itemsInTransitReport.size(), is(200));
   }
 
+  @Test
+  void reportShouldNotFailWithoutLastServicePointId() {
+    ItemResource item = checkOutAndCheckInItem();
+
+    Response response = itemsClient.getById(item.getId());
+    JsonObject checkedInItemJson = response.getJson();
+    checkedInItemJson.getJsonObject("lastCheckIn").remove("servicePointId");
+    itemsClient.replace(item.getId(), checkedInItemJson);
+
+    List<JsonObject> itemsInTransitReport = ResourceClient.forItemsInTransitReport().getAll();
+
+    assertThat(itemsInTransitReport.size(), is(1));
+  }
+
+  @Test
+  void reportShouldNotFailWithoutPrimaryServicePointId() {
+    ItemResource item = checkOutAndCheckInItem();
+
+    Response response = itemsClient.getById(item.getId());
+    JsonObject checkedInItemJson = response.getJson();
+    UUID permanentLocationId = UUID.fromString(checkedInItemJson.getString("permanentLocationId"));
+    JsonObject location = locationsClient.getById(permanentLocationId).getJson();
+    location.put("primaryServicePoint", null);
+    locationsClient.replace(permanentLocationId, location);
+
+    List<JsonObject> itemsInTransitReport = ResourceClient.forItemsInTransitReport().getAll();
+
+    assertThat(itemsInTransitReport.size(), is(1));
+  }
+
+  private ItemResource checkOutAndCheckInItem() {
+    final UUID firstServicePointId = servicePointsFixture.cd1().getId();
+    final UUID forthServicePointLocationId = locationsFixture.fourthServicePoint().getId();
+
+    ItemResource item = createSmallAngryPlanetCopy(forthServicePointLocationId, "111");
```

Comment:
create item without parameters, check in at service point != cd1

Reply:
it doesn't work without parameters, we should specify a service point